### PR TITLE
[develop2] warning for duplicated URL remote

### PR DESCRIPTION
**Pull request by memsharded**
Changelog: omit

Close https://github.com/conan-io/conan/issues/1430

### CODE REVIEW

File: conans/test/integration/command/remote_test.py [hunk 1]
```diff
@@ -278,3 +278,16 @@ def test_invalid_url(self):
                       self.client.out)
         self.client.run("remote list")
         self.assertIn("pepe.org", self.client.out)
+
+
+def test_duplicated_url():
+    """ allow duplicated URL with --force
```
Comment (AbrilRBS) [inline conans/test/integration/command/remote_test.py:284]: @memsharded this comment implies `--force` would be needed for this to work, but I do not see such either in the commands below or the related code. Is this a bug or a typo?

Comment (memsharded) [reply conans/test/integration/command/remote_test.py:284]: You are right, the comment is obsolete. No --force is necessary, it was replaced with a warning. Do you think that we should change it to ``--force``?

Comment (AbrilRBS) [reply conans/test/integration/command/remote_test.py:284]: That would certainly be my expectation for this, yes :)

Comment (memsharded) [reply conans/test/integration/command/remote_test.py:284]: Sure, yes, I agree, lets change it

Comment (memsharded) [reply conans/test/integration/command/remote_test.py:284]: https://github.com/conan-io/conan/pull/13112

**Pull request merged**